File: api/cas-management-api/src/main/java/org/apereo/cas/mgmt/domains/Commit.java
package org.apereo.cas.mgmt.domains;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * Bean to serialize a commit to the client.
 *
 * @author Travis Schmidt
 * @since 5.2.0
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Commit {
    /**
     * Id of the commit.
     */
    private String id;

    /**
     * Commit message of the commit.
     */
    private String text;

    /**
     * Time commit occurred.
     */
    private String commitTime;
}

File: webapp/src/domain/commit.ts
/**
 * Created by tsschmi on 3/8/17.
 */
export class Commit {
  id: String;
  text: String;
  commitTime: String;

  constructor(id: String, text: String, commitTime: String) {
    this.id = id;
    this.text = text;
    this.commitTime = commitTime;
  }
}
